index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8" />
+    <title>TypingPractice</title>
+    <link rel="stylesheet" href="style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  </head>
+  <body>
+    <div class="wrapper" id="game">
+      <input type="text" class="input-field" />
+      <div class="content-box">
+        <input type="number" placeholder="Generated Words" id="word_count" max="25" onKeyDown="return false">
+        <button onclick="generateWords()" >Generate</button>
+        <div class="typing-text">
+          <p id="paragraph"></p>
+        </div>
+        <div class="content">
+          <ul class="result-details">
+            <li class="time">
+              <p>Time Left:</p>
+              <span><b>60</b>s</span>
+            </li>
+            <li class="mistake">
+              <p>Mistakes:</p>
+              <span>0</span>
+            </li>
+            <li class="wpm">
+              <p>WPM:</p>
+              <span>0</span>
+            </li>
+            <li class="cpm">
+              <p>CPM:</p>
+              <span>0</span>
+            </li>
+          </ul>
+          <button>Try Again</button>
+        </div>
+      </div>
+    </div>
+
+    <div class="wrapper" id="scoreboard">
+      <div class="content-box">
+        <p>Time is up!</p>
+        <br />
+        <p>Your Total Best</p>
+        <div class="best-score-container">
+          <div id="left">Mistakes: <span id="alltime_mistakes">0</span></div>
+          <div>WPM: <span id="alltime_wpm">0</span></div>
+          <div id="right">CPM: <span id="alltime_cpm">0</span></div>
+        </div>
+        <br />
+        <p>Your todays Best</p>
+        <div class="best-score-container">
+          <div id="left">Mistakes <span id="today_mistakes">0</span></div>
+          <div>WPM: <span id="today_wpm">0</span></div>
+          <div id="right">CPM: <span id="today_cpm">0</span></div>
```

Comment:
```suggestion
          <div id="right">CPM: <span id="today-cpm">0</span></div>
```

Also ids have to be unique.